docs/common_issues.rst

```diff
@@ -42,8 +42,28 @@ can add `changeReason` on each instance:
     >>> Poll.history.get(id=data[0].id).history_change_reason
     'reason'
 
+Bulk Updating a Model with History (New)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Bulk update was introduced with Django 2.2. We can use the utility function
+``bulk_update_with_history`` in order to bulk update objects using Django's ``bulk_update`` function while saving the object history:
+
+.. _bulk_create: https://docs.djangoproject.com/en/2.0/ref/models/querysets/#bulk-create
+
+
+.. code-block:: pycon
+
+    >>> from simple_history.utils import bulk_create_with_history
+    >>> from simple_history.tests.models import Poll
+    >>> from django.utils.timezone import now
+    >>>
+    >>> data = [Poll(id=x, question='Question ' + str(x), pub_date=now()) for x in range(1000)]
+    >>> objs = bulk_create_with_history(data, Poll, batch_size=500)
+    >>> for obj in objs: obj.question = 'Duplicate Questions'
+    >>> bulk_update_with_history(objs, Poll, ['question'], batch_size=500)        
+    >>> Poll.objects.first().question
+    'Duplicate Question'
 
-QuerySet Updates with History
+QuerySet Updates with History (Updated in Django 2.2) 
```

Comment:
extend the underline under this all the way. if you run `make docs` locally, you'll see the warning that come up if you don't do this